start.py

```
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
```

Ruff (0.11.9)

Shebang is present but file is not executable

(EXE001)